src/common/types/navigation/rootStackParamList.ts

```diff
@@ -11,6 +11,7 @@ type RootStackParamList = {
     playback?: boolean
   }
   [NavigationScreen.TO_TABS_NAVIGATOR]: undefined
+  [NavigationScreen.MY_PROFILE]: undefined
```

Comment:
Let's remove part `MY_`
```suggestion
  [NavigationScreen.PROFILE]: undefined
```